recipe/meta.yaml

```diff
@@ -159,21 +166,21 @@ outputs:
         - psycopg2 =2
         - pyarrow
         - pybind11
-        - pydantic <2
+        - pydantic >=2,<3
         - pyld
         - pysynphot
         - pytables
         - python
         - pytorch
         - pyyaml
         - requests
-        - ruff ==0.0.275
+        - ruff ==0.0.278
```

Comment:
0.1.7?

Reply:
There wasn't an RFC/ticket for it, but sure.